force-di/main/classes/di_PlatformCache.cls

```diff
@@ -64,40 +68,176 @@ public with sharing class di_PlatformCache
 
             if ( cacheKeyIndexMap == null )
             {
-                cacheKeyIndexMap = new Map<String, Map<Schema.SObjectType, Set<String>>>();
+                cacheKeyIndexMap = new Map<String, Map<Schema.SObjectType, Set<String> > >();
             }
-            
+
             /*
```

Comment:
Should we be keeping commented code? I vote to remove it.

Reply:
@stohn777
It seemed to me that there is no more issue for this one above. Correct?